src/request/export/ExportFile.js

```diff
@@ -95,7 +97,11 @@ class ExportFile extends Nimiq.Observable {
         });
 
         this._passwordBox.on(PassphraseBox.Events.SUBMIT, async password => {
-            await this._passwordSubmitted(password);
+            if (this._key && this._password) {
+                await this._passwordSubmitted(this._password);
+            } else {
+                await this._passwordSubmitted(password);
```

Comment:
Does this make sense to separate those now? If the passphrase is entered, why would you use another password then? Even when the password is right there as a parameter?

Reply:
Because it is not. The PassphraseBox can `.hideInput()`, but the button then still fires `PassphraseBox.Events.Submit`. However, there is no input.

Reply:
Ah. So does it maybe make sense to do the check the other way around (check if `password` and only then do the check for _key and _password, including a comment of when this situation might happen?

Reply:
`undefined` is in general a valid password. So doing it the other way around is not a good idea I think, even though it would work in this case.
I will add a comment about it.

Reply:
Apparently no comment was added :laughing:

Reply:
Apparently I forgot about it. Will add it in the current feedback branch.